pkg/resources/grant_privileges_to_account_role_acceptance_test.go

```diff
@@ -0,0 +1,909 @@
+package resources_test
+
+import (
+	"context"
+	"database/sql"
```

Comment:
the tests look similar to database role ones, I have to admit I skimmed through them. They look fine from the names I did not validate in detail the checks